apps/myaccount/src/hooks/use-sign-in.ts

```diff
@@ -0,0 +1,441 @@
+                        window["AppUtils"].getConfig().superTenant,
+                        window["AppUtils"].getConfig().superTenantProxy
+                    );
+                }
+
+                // If `oidc session iframe` endpoint is overridden, save that in the session.
+                if (window["AppUtils"].getConfig().idpConfigs?.oidcSessionIFrameEndpointURL) {
+                    oidcSessionIframeEndpoint = resolveIdpURLSAfterTenantResolves(
+                        oidcSessionIframeEndpoint,
+                        window["AppUtils"].getConfig().idpConfigs.oidcSessionIFrameEndpointURL
+                    );
+                }
+
+                // If `token` endpoint is overridden, save that in the session.
+                if (window["AppUtils"].getConfig().idpConfigs?.tokenEndpointURL) {
+                    tokenEndpoint = resolveIdpURLSAfterTenantResolves(
+                        tokenEndpoint,
+                        window["AppUtils"].getConfig().idpConfigs.tokenEndpointURL
+                    );
+                }
+
+                sessionStorage.setItem(AUTHORIZATION_ENDPOINT, authorizationEndpoint);
+                sessionStorage.setItem(OIDC_SESSION_IFRAME_ENDPOINT, oidcSessionIframeEndpoint);
+                sessionStorage.setItem(TOKEN_ENDPOINT, tokenEndpoint);
+
+                // Update post_logout_redirect_uri of logout_url with tenant qualified url
+                const logoutUrl: string = window["AppUtils"].getConfig().idpConfigs?.logoutEndpointURL;
+                let logoutRedirectUrl: string;
+
+                updateConfig({
+                    endpoints: {
+                        authorizationEndpoint: authorizationEndpoint,
+                        checkSessionIframe: oidcSessionIframeEndpoint,
+                        endSessionEndpoint: logoutUrl.split("?")[0],
+                        tokenEndpoint: tokenEndpoint
+                    },
+                    signOutRedirectURL: deriveLogoutRedirectForSubOrgLogins(
+                        logoutRedirectUrl,
+                        idToken.user_org,
+                        idToken.org_id
+                    )
+                });
+            })
+            .catch((error: AsgardeoAuthException) => {
+                throw error;
+            });
+
+        dispatch(getProfileInformation());
+    };
+
+    /**
+     * Handles the sign-in process for legacy authorization server.
+     * @deprecated This is deprecated and will be removed in the next major release.
+     * @param response - The basic user information returned from the sign-in process.
+     */
+    const legacyOnSignIn = async (response: BasicUserInfo): Promise<void> => {
+        let logoutUrl: string;
+        let logoutRedirectUrl: string;
+
+        const event: Event = new Event(CommonConstantsCore.AUTHENTICATION_SUCCESSFUL_EVENT);
+
+        dispatchEvent(event);
+
+        let tenantDomain: string = AuthenticateUtils.deriveTenantDomainFromSubject(response.sub);
```

Comment:
Shall we make this const since it's not being reassigned later in the function?

Reply:
It was reassigned in a later step but figured out that we can merge the reassignment.

Fixed in Fixed with https://github.com/wso2/identity-apps/pull/5141/commits/cec412cdc4921989a3ac1867458613650b88e620